Anime-App/Feature/Anime/URL/URL.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  URL.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 12/08/2023.
+//
+
+import Foundation
+
+enum UrlEndpoint {
+    case baseUrl
+    case getAnime
+    case searchAnime
+}
+
+extension UrlEndpoint {
+    var endpoint: String {
+        switch self {
+        case .baseUrl:
```

Comment:
So the base url shouldn't be an enum case it should be separate in a separate enum, a base URL is not the same as an endpoint so separate these two